plugins/out_cloudwatch_logs/cloudwatch_logs.c

```diff
@@ -162,6 +162,13 @@ static int cb_cloudwatch_init(struct flb_output_instance *ins,
         ctx->create_group = FLB_TRUE;
     }
 
+    ctx->retry_requests = FLB_FALSE;
+    tmp = flb_output_get_property("auto_retry_requests", ins);
+    /* follow precedent */
```

Comment:
nit: Just comment on what this line does: 
```
/* native plugins use On/Off as bool, the old Go plugin used true/false */
```

And actually this feature is for C plugin only now so maybe it is not applicable to Go plugin. But you could keep it once we would like to extend it to our Go plugin.

Reply:
Done